src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -119,6 +125,31 @@ static MvcCoreLoggerExtensions()
                 2,
                 "Executed action method {ActionName}, returned result {ActionResult}.");
 
+            _logFilterExecutionPlan = LoggerMessage.Define<string, string[]>(
+                LogLevel.Debug,
+                1,
+                "Execution plan of {FilterType} filters (in the following order): {Filters}");
+
+            _beforeExecutingMethodOnFilter = LoggerMessage.Define<string, string, string>(
+                LogLevel.Trace,
+                2,
+                "{FilterType}: Before executing {Method} on filter {Filter}.");
+
+            _afterExecutingMethodOnFilter = LoggerMessage.Define<string, string, string>(
+                LogLevel.Trace,
+                3,
+                "{FilterType}: After executing {Method} on filter {Filter}.");
```

Comment:
I'm not a super big fan of the fact that all of the filter types have them same log ids